hr_employee_calendar_planning/models/hr_employee.py

```diff
@@ -117,6 +109,13 @@ def _regenerate_calendar(self):
     def regenerate_calendar(self):
         self._regenerate_calendar()
 
+    @api.model_create_multi
+    def create(self, vals_list):
```

Comment:
Why this extra `create` overriding?

Reply:
This is necessary because if calendars are defined in the creation of the employee, the default calendar of the company is defined as the employee calendar (`resource_calendar_id` field), and this is incorrect, the automatic employee calendar should be auto-generated. The `test_employee_with_calendar_ids()` test validates just this.